packages/frint-cli/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
is it possible to have one single `tsconfig.json` in the root of the repository?

Reply:
I looked into it: there is a way to have a base config, and then extend it, but it's not working to me properly.  
It's already a bit awkward to use, because all the paths in it will be relative to where the actual file is, so we can't have things like `"output": "./lib"` there, since it'll be relative to the root, so those fields would have to be overriden in every package anyway.  
But even this way it didn't work for me, tsc produced some weird compilation error about not finding d.ts files, I think also related to relative paths.

So for now I'll leave as it is, but maybe we'll find a way to unify them.